ArduCopter/mode_plancktracking.cpp

```diff
@@ -193,6 +193,15 @@ void ModePlanckTracking::run() {
       }
     }
 
+    if (g.failsafe_planck_angle > 0){
```

Comment:
Should the failsafe check be called from the mode? I thought the `Copter` had its own failsafe check call?

Reply:
I went back and forth on this. There seems to be a few ways failsafe checks are called, and a several different ways failsafes are implemented. I could make it a scheduled task instead, but I was just trying to keep it a simple as possible. Shall I go ahead and make it a task?